packages/react-select/src/Select.js

```diff
@@ -1917,14 +1788,32 @@ export default class Select extends Component<Props, State> {
   }
 
   renderLiveRegion() {
-    if (!this.state.isFocused) return null;
+    const { commonProps } = this;
+    const {
+      ariaSelection,
+      focusedOption,
+      focusedValue,
+      isFocused,
+      selectValue,
+    } = this.state;
+
+    const { inputValue, menuIsOpen, options } = this.props;
+
+    const focusableOptions = menuIsOpen ? this.getFocusableOptions() : [];
```

Comment:
`getFocusableOptions` already checks `menuIsOpen`, so you should be able to just call `this.getFocusableOptions()`.

Reply:
I meant to also ask here if this is causing any additional overhead by calling `getFocusableOptions` which in turn calls `buildFocusableOptions`? When I originally was working on this, focusableOptions were in state.

Reply:
Not overhead that doesn't already exist.